### ISSUE TITLE
silent ignore of list metrics

### ISSUE THREAD

**Issue by lakshya-4gp** [aimhubio/aim#2558]
## 🐛 Bug

If Run.track receives a list input, it is silently ignored

### To reproduce

>>> import aim
>>> run = aim.Run()
>>> x = []
>>> run.track(x, name="temp")

UI doesn't show any metrics in the corresponding run

### Expected behavior

I believe at least a warning, but personally a error should be thrown

### Environment

- Aim Version (e.g., 3.0.1) 3.16.1
- Python version 3.9.16
- pip version N/A
- conda version 23.1.0
- OS (e.g., Linux) Linux
- Any other relevant information

**Comment by alberttorosyan** [comment]
Hey @lakshya-4gp! Thanks for reporting this.

Aim allows to track different types of metadata, where the Metric is one of them. In order to display the tracked sequence in metrics UI, its element type should be a numeric type (either integer or floating point).

**Comment by lakshya-4gp** [comment]
Fair enough, I guess it can be decided b/w when the user wants to log some list vs when the user made the mistake of thinking they are logging a scaler but not a list. Perhaps, somewhere in the doc, it should be highlighted that UI will ignore the logged list values.
